firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -337,7 +337,7 @@ private PersistedInstallationEntry getPrefsWithGeneratedIdMultiProcessSafe() {
   }
 
   /** Use file locking to acquire a lock that will also block other processes. */
-  private FileLock getCrossProcessLock() {
+  FileLock getCrossProcessLock() {
```

Comment:
Why remove the private here?

I don't see any cross-class references added in this PR.